libs/labelbox/tests/data/annotation_import/test_data_types.py

```diff
@@ -1,12 +1,8 @@
 import datetime
```

Comment:
Heavily reduced the test on this file and focused on unit testing serializing data type labels to NDJSON format. Generic data types tests the importing of NDJSON

Reply:
Also data types is marked as depercated